bigtop.bom

```diff
@@ -67,7 +67,7 @@
 
 bigtop {
 /** Base Configuration of the mirror and archives */
-  version = "1.2.0-SNAPSHOT"
+  version = "1.1.0-SNAPSHOT"
```

Comment:
I am not sure how that got changed, must have been by mistake as I don't see any reason to change it to 1.1.0, will fix that. thanks for pointing out.